### PR TITLE
IM-36 Add Product Form

### PR DESCRIPTION
**Pull request by claudiulodro**
This is the Create Product form and some re-usable responsive styles that we can use on other input forms too.

### CODE REVIEW

File: views/NewProduct.html [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+            var product_name = $( '.product-name').val();
+            var description = $( '.description').val();
+            var date = getDate();
+
+            if( !customer.length || !product_name.length || !description.length ){
+                submit_button.after( "<div class='error-message'>One or more required fields isn\'t filled out</div>" );
+                return;
+            }
+
+            host = host + "/"
```
Comment (hinyunlee) [inline views/NewProduct.html]: The address (host) already has a slash at the end.

File: views/NewProduct.html [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+                    + date + "/";
+
+            $.get( host, function( response ) {
+
+                if( response == "Success" ){
+                    submit_button.after( "<div class='success'>Successfully created product</div>" );
+                    submit_button.remove();
+                }
+                else {
+                    submit_button.after( "<div class='error-message'>" + response + "</div>");
```
Comment (hinyunlee) [inline views/NewProduct.html:121]: No error message is shown when there is no response (ie. http 404)

**Comment by hinyunlee** [comment]
The code looks great. I have commented on some minor issues.

**Comment by claudiulodro** [comment]
I've addressed these issues in the most recent commit.

**Pull request merged**